source/our_tools/tractoflow.rst

```diff
@@ -3,4 +3,220 @@
+
+Installing Tractoflow
+*********************
+
+    You can check the `installation guide <https://tractoflow-documentation.readthedocs.io/en/latest/installation/before_install.html>`_ on Tractoflow's website (pages before install and install). If you want to install it on Compute Canada, connect first and follow instructions in the High Performance Computer sections.
+
+Using Tractoflow on Compute Canada
+**********************************
+
+    To learn about Compute Canada, please see the :ref:`ref_heavy_computing` page.
+
+    To use tractoflow on a computing platform, you will find some information on Tractoflow's website, section `How to launch tractoflow <https://tractoflow-documentation.readthedocs.io/en/latest/pipeline/launch.html>`_. Here is additional, more complete instructions:
+
+    #. Bring data to your cluster. Ex, here, on Beluga.
+
+        .. code-block:: bash
+
+            scp MY_DATA.zip USER@beluga.computecanada.ca:
+            scp -r MY_DIR USER@beluga.computecanada.ca:  # To send a whole folder, unzipped.
+
+    #. Connect to the cluster using ssh. Type ls and make sure that you see
+
+        | tractoflow-x.x.x/
+        | tractoflow_x.x.x_version_20xx-xx-xx.img
+        | MY_DATA.zip
+
+    #. Bring your data to your scratch directory.
+
+        .. code-block:: bash
+
+            mkdir scratch/Project_X
+            mv MY_DATA.zip scratch/Project_X/
+            cd scratch/Project_X/
+            unzip MY_DATA.zip
+            rm -rf MY_DATA.zip
+
+    #. You now have all your input data ready to go for tractoflow. If you want to double-check that all the necessary files are there. Run
+
+        .. code-block:: bash
+
+            for a in input_data_tractoflow/*; do ls $a; done
+
+       You should see bval  bvec  dwi.nii.gz  t1.nii.gz, as many times as you have subjects.
+
+    #. Nextflow will need to find your data in the scratch. As explained on Tractoflow's website (section How to launch your data), create a file beluga.conf in your project (ex, using nano), which contains these 2 lines.
+
+        .. code-block:: bash
+
+            singularity.runOptions="--bind /lustre04/scratch/USER/"
+            process.executor='ignite'
+
+
+    #. If you want to have an idea of the b-values in your protocol because you are not sure (this is useful to know to set the dti_shells and fodf_shells in your future tractoflow command):
+
+        .. code-block:: bash
+
+            for a in input_data_tractoflow/*; do cat $a/bval; done
+
+    #. Tractoflow is 100% reproducible if you re-use in the future the SAME code and SAME container image (singularity). Hence, copy them locally in your project to SAVE them in the future.
```

Comment:
Don't like that. I prefer to have everything in the same folder like:

My_project:
-> tractoflow_*.img
-> tractoflow-*/
-> results/
-> work/
...

In my opinion, moving in an other location is the best thing to forget to update them if you change versions in your processing location.

Reply:
How about if you don't want to leave everything in the scratch to avoid having everything deleted?

Maybe I should keep this but as a last step and add cp scratch to project?

Reply:
Just copy everything but not the work. Or if you are sure, copy just the useful folders in the results and everything else (img, tractoflow etc..)

Reply:
Ok. See the updated last point.